.github/workflows/build-continuous-linux.yml

```diff
@@ -10,19 +9,20 @@ jobs:
       - uses: actions/checkout@v2.3.4
       - name: Install Qt5
         run: |
-          sudo apt-get install -y build-essential gettext qttools5-dev libqt5svg5-dev libqt5xmlpatterns5-dev
+          # register kde neon repository
+          echo 'deb http://archive.neon.kde.org/user focal main' | sudo tee /etc/apt/sources.list.d/neon.list
+          curl https://origin.archive.neon.kde.org/public.key -o - | sudo apt-key add -
+          
+          # install build dependencies
+          sudo apt-get update
+          sudo apt-get install -y build-essential gettext git qt5-default qttools5-dev libqt5svg5-dev libqt5xmlpatterns5-dev
```

Comment:
Isn't qt5-default on focal qt 5.12.5?

Reply:
Yes I know my Seamly2d.pro sets the QT_VERSION to 5.13.2. 
Do I need to edit this NOW to 5.15.2?

Reply:
KDE NEON usually update the Qt version to the very latest if possible. We may need to find another source if we want to fix (which I think is a good idea) version.

Reply:
> Do I need to edit this NOW to 5.15.2?

If your source code doesn't depend on features specific to 5.15.2 you can leave 5.13.2 as required Qt version.